### DISCUSSION TITLE
How do you make an instance?

### DISCUSSION THREAD

**Discussion by Apixirli** [Autodesk/maya-usd#1467] [Q&A] [answered]
Not point instancing, just plain old instances. I know USD supports them but not quite sure how they work. Maya's duplicate special instances don't work. I assume just duplicating the prims would convert them to instances somehow when USDStage analyzes the file and creates a prototype for it?

What I read: [https://graphics.pixar.com/usd/docs/api/_usd__page__scenegraph_instancing.html](https://graphics.pixar.com/usd/docs/api/_usd__page__scenegraph_instancing.html)

**Comment by Apixirli** [comment, marked as answer]
Figured out that making instances while in Maya's data model then exporting it to USD converts them. I suppose there's no way of making instances while in USD in Maya at the moment?

**Reply by pberto** [reply]
Multiverse :)

**Comment by wtelford** [comment]
It isn't yet a one-click solution but with MayaUSD 0.9.0 you do have access to editing metadata in the Attribute Editor
You can build the "Explicit Instances, Implicit Prototypes" example on this page
https://graphics.pixar.com/usd/docs/api/_usd__page__scenegraph_instancing.html

1. Create a new stage with Create > USD > Stage With New Layer
2. In Outliner right-click the stageShape > Add New Prim > Def
3. Right click Def1 > Add Reference, then select your usd file you want to reference
4. Open the AE for Def1 and set the instanceable flag in the metadata section
5. With Def1 selected in the viewport hit ctrl-D to duplicate it.

**Reply by adamdewhirst** [reply]
with you so far Will - up to point 4, not seeing where the metadata section is - any chance of a screen grab?

**Comment by wtelford** [comment]
Hey @adamdewhirst,
Someone followed up with me and made me aware this wasn't answered. 
![image](https://user-images.githubusercontent.com/1900693/144873592-56d81894-26f3-4174-bec5-61f3c64c76d8.png)
